src/System.Globalization/tests/Performance/Perf.StringCultureInfo.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I see. I was looking at https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/String.cs to determine if they were present, so that makes sense. Will remove them.